packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -626,6 +627,23 @@ void main() {
   }, overrides: <Type, Generator>{
     FeatureFlags: () => TestFeatureFlags(isWebIncrementalCompilerEnabled: true),
   }));
+
+  test('connect sets up log reader', () => testbed.run(() async {
+    final MockDevice mockDevice = MockDevice();
+    final MockDeviceLogReader mockLogReader = MockDeviceLogReader();
+    when(mockDevice.getLogReader(app: anyNamed('app'))).thenReturn(mockLogReader);
+
+    final TestFlutterDevice flutterDevice = TestFlutterDevice(
+      mockDevice,
+      <FlutterView>[],
+      observatoryUris: <Uri>[ testUri ]
+    );
+
+    await flutterDevice.connect();
+    verify(mockLogReader.connectedVMServices = <VMService>[ mockVMService ]);
```

Comment:
wow, does this verify that an assignment happened? I didn't know you could do that.

Reply:
Seems to.  From the mockito docs:
```
// You can verify setters.
cat.lives = 9;
verify(cat.lives=9);
```

Reply:
I've skimmed that README many times and yet seem to have missed that. Cool.